catalog/video/video-admin-plugin/src/main/java/org/codice/alliance/video/ui/service/StreamMonitorHelper.java

```diff
@@ -49,6 +55,25 @@
 
     private static final String ID = "id";
 
+    private static final Predicate<InetAddress> IPV4_FILTER =
+            inetAddress -> inetAddress instanceof Inet4Address;
+
+    private static final Predicate<NetworkInterface> INTERFACE_WITH_IPV4 =
+            networkInterface -> Collections.list(networkInterface.getInetAddresses())
+                    .stream()
+                    .anyMatch(IPV4_FILTER);
+
+    private static final Predicate<NetworkInterface> SUPPORTS_MULTICAST = networkInterface -> {
```

Comment:
I usually choose the approach that I find more readable. I suppose I would choose a method if it needed to access some other member variable or method of the class, and choose a constant if it's more self-contained.